src/ServiceControl.UnitTests/Infrastructure/Extensions/NegotiatorExtensionsTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿namespace ServiceControl.UnitTests.Infrastructure.Extensions
+{
+    using System;
+    using System.Linq;
+    using Nancy;
+    using Nancy.Responses.Negotiation;
+    using NUnit.Framework;
+    using ServiceBus.Management.Infrastructure.Extensions;
+
+    [TestFixture]
+    public class NegotiatorExtensionsTests
+    {
+        [TestCase]
+        public void WithPagingLinks_ReturnsLinksWithRelativeUri()
+        {
+            var pagingHeaders = GetLinks(200, currentPage: 3, path: "/api/test");
```

Comment:
I would prefer to use named parameters in all the test to make it a bit more readable.

Reply:
@danielmarbach you mean for the first parameter?

Reply:
yes